cmd/preview/list_test.go

```diff
@@ -142,3 +148,41 @@ func Test_PreviewListOutputValidation(t *testing.T) {
 		})
 	}
 }
+
+func Test_getPreviewDefaultOutput(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    previewOutput
+		expected string
+	}{
+		{
+			name: "preview with no labels",
+			input: previewOutput{
+				Name:     "my-preview",
+				Scope:    "personal",
+				Sleeping: false,
+			},
+			expected: "my-preview\tpersonal\tfalse\t-\n",
+		},
+		{
+			name: "preview with labels",
+			input: previewOutput{
+				Name:     "my-preview",
+				Scope:    "personal",
+				Sleeping: false,
+				Labels: map[int]string{
```

Comment:
Run tests several times because it can happen that the order is not the same. Check https://go.dev/blog/maps `Iteration order`